api/app/signals/apps/api/serializers/signal.py

```diff
@@ -542,6 +543,30 @@ def get__display(self, obj):
         return obj.sia_id
 
 
+class PublicSignalGeographtSerializer(serializers.Serializer):
```

Comment:
In general I am wondering why you did not added the checks to the filter? I think there are ways to make that happen and it would be more clear than an extra serializer.